pwiz_tools/Skyline/Skyline.csproj

```diff
@@ -5289,162 +5293,162 @@
   <ItemGroup>
     <PublishFile Include="BiblioSpec.pdb">
       <Visible>False</Visible>
+      <PublishState>Include</PublishState>

```

Comment:
Revert all these Include, True changes to avoid fighting over this ordering with other developers.